popde/density_estimate.py

```diff
@@ -256,8 +268,8 @@ def evaluate_with_transf(self, points):
         # Jacobian for training data standardization
         std_Jacobian = 1. / np.prod(self.stds) if self.stdize else 1.
 
-        # Jacobian for rescaling factor
-        rescale_Jacobian = 1. / np.prod(self.rescale) if (self.rescale is not None) else 1.
+        # Jacobian for rescaling factor f(x) = f(x'*c) = f(x')*c for each dim
```

Comment:
The derivation here seems wrong. 
If F(x') is the KDE for rescaled variables and if x = x' * c, then f(x) is F(x') * c^-1. 

Recall that $\int f(x) dx$ has to be equal to $\int F(x') dx'$.  If the variables get larger the KDE has to get smaller.

Reply:
fixed now